src/app/components/todostatus/todostatus.component.ts

```diff
@@ -20,14 +20,21 @@ export class TodostatusComponent implements OnInit, OnDestroy {
     this.countSubscription = this.todolistComponent.countEmitter.subscribe(
       count => {
         this.taskCompleted = count;
+        if(this.taskCompleted==0){
```

Comment:
it should be true/false not 1 or 0

Reply:
we can use true and false but here we use task completed as a number